spec.html

```diff
@@ -1499,7 +1501,7 @@ <h1>Number::bitwiseNOT ( _x_ )</h1>
           <p>The abstract operation Number::bitwiseNOT takes argument _x_ (a Number). It performs the following steps when called:</p>
           <emu-alg>
             1. Let _oldValue_ be ! ToInt32(_x_).
-            1. Return the result of applying bitwise complement to _oldValue_. The result is a signed 32-bit integer.
+            1. Return the result of applying bitwise complement to _oldValue_. The result is a signed 32-bit integral Number.
```

Comment:
I don't know if we say anywhere what it means to be "a signed 32-bit integral Number". I would just drop the sentence.

```suggestion
            1. Return the result of applying bitwise complement to _oldValue_.
```

Reply:
For now, I removed them.